tests/cli/test_main_clean.py

```diff
@@ -187,7 +187,7 @@ def test_clean_tempfiles(clear_cache):
 def test_clean_logfiles(clear_cache):
```

Comment:
Do we need to remove this test or are we leaving it here so people can remove old logfiles from previous installations?